distributed/tensor_parallelism/tensor_parallel_example.py

```diff
@@ -1,11 +1,17 @@
 import argparse
-
+import os
 import torch
-import torch.multiprocessing as mp
 
-from torch.distributed._tensor import DeviceMesh
-from torch.distributed.tensor.parallel import PairwiseParallel, parallelize_module
-from utils import cleanup, setup, ToyModel
+from torch.distributed._tensor.device_mesh import init_device_mesh
+from torch.distributed._tensor import DeviceMesh, DTensor, Replicate, Shard
+
+from torch.distributed.tensor.parallel import (
+    PairwiseParallel,
+    parallelize_module,
+    ColwiseParallel,
+    RowwiseParallel,
+)
+from utils import cleanup, ToyModel, torchrun_setup
```

Comment:
this file I have exactly same comments from `sequence_parallel_example.py`, please address accordingly :)

Reply:
all removed :)